docker/hq-compose.yml

```diff
@@ -79,13 +80,21 @@ services:
         hard: 1048576
     expose:
       - "5984"
+    healthcheck:
+      test: bash -c 'exec 6<>/dev/tcp/couch/5984'
```

Comment:
The default shell is sh, which does not support this. Explicitly use bash.

This is the method used by wait.sh and is lighter than using netcat.